source/renderer/app/actions/daedalus-diagnostics-actions.js

```diff
@@ -3,7 +3,7 @@ import Action from './lib/Action';
 
 // ======= NETWORK STATUS ACTIONS =======
 
-export default class NetworkStatusActions {
+export default class DaedalusDiagnosticsActions {
```

Comment:
@DeeJayElly please revert the name of the actions to `NetworkStatusActions` 🙏

Reply:
@nikolaglumac done